### PR TITLE
Create annoucement and category models

### LINKED ISSUE #4: Annoucement model

### LINKED ISSUE #5: Category model

### PR DESCRIPTION
**Pull request by zawolanski**
I created:
- annoucement model
- annoucement query
- category model
- category query
- getCategory query - you have to pass `categoryId` as parameter

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -40,21 +43,10 @@ const app = express();
 //   multer({ storage: fileStorage, fileFilter: fileFilter }).single('image')
 // );
 // app.use('/images', express.static(path.join(__dirname, 'images')));
```
Comment (pawelsobkowski) [inline app.js:45]: Please, remove comments if not needed.

File: app.js [hunk 2]
```diff
@@ -40,21 +43,10 @@ const app = express();
...
-  if(req.method === 'OPTIONS') {
-    return res.sendStatus(200)
-  }
-  next();
-});
-
-app.use(auth)
+app.use(auth);
 
 // app.put('/add-image', (req, res, next) => {
```
Comment (pawelsobkowski) [inline app.js:51]: Same as above.

File: app.js [hunk 3]
```diff
@@ -104,17 +96,19 @@ app.use((error, req, res, next) => {
...
+  .catch((err) => console.log(err));
 
 // const clearImage = filePath => {
 //   filePath = path.join(__dirname, '..', filePath);
 //   fs.unlink(filePath, err => console.log(err));
 // };
 
-//r6MsmhxSbo5yRgPF
\ No newline at end of file
+//r6MsmhxSbo5yRgPF
```
Comment (pawelsobkowski) [inline app.js]: Is it necessary?

**Review by pawelsobkowski — COMMENTED** [review]
Code looks good, but I left a comment about comments left

File: app.js (hunk 1 above)
Comment (lzawistowsky) [reply app.js:45]: i need that for image upload

**Review by lzawistowsky — APPROVED** [review]
ok

File: app.js (hunk 3 above)
Comment (zawolanski) [reply app.js]: I don't know because i didn't leave this comment, but i will remove it.

File: app.js (hunk 1 above)
Comment (zawolanski) [reply app.js:45]: I kept it because i didn't leave this comments.

**Review by pawelsobkowski — APPROVED** [review]

**Pull request merged**